atlas/AtlasMl/tests/test_pipelines_unit.py

```
@@ -1,60 +1,67 @@
+        mock_embed.side_effect = lambda embedding: (uuid, [0.5, 0.5])
         workflows.initial_competencies(competencies)
     assert workflows.weaviate_client.add_embeddings.call_count == 2
     calls = workflows.weaviate_client.add_embeddings.call_args_list
     for c, comp in zip(calls, competencies):
         assert c[0][0] == "Competency"
-        assert c[0][2]["name"] == comp["title"]
-        assert c[0][2]["text"] == comp["description"]
+        assert c[0][2]["title"] == comp.title
+        assert c[0][2]["description"] == comp.description
 
 def test_initial_cluster_to_competencyPipeline(workflows):
```

Ruff (0.12.2)

Function name `test_initial_cluster_to_competencyPipeline` should be lowercase

(N802)